tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Install openvpn
+  apt: 
+    name: openvpn
+
+- name: Slurp openvpn config
+  slurp:
+    src: "{{ hostvars[openvpn_client_server].openvpn_base_dir }}/{{ openvpn_client_name }}-{{ openvpn_client_server }}.ovpn"
+  register: openvpn_config_file
+  delegate_to: "{{ openvpn_client_server }}"
+  when: openvpn_client_server is defined
+
+- name: Copy to host
```

Comment:
Copy OpenVPN config to host

same below

Reply:
fixed